CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 3.1.1
+
+- Make `isInKeepAlive` on `SseConnection` private.
+
+* Note that this is a breaking change but in actuality no one should be
```

Comment:
This renders as 2nd bullet list which is probably not what we want. Did you intend for a nested list or something else?

Reply:
Intended to have two `**`.

Reply:
I think that renders literally... If you add trailing ones too it would make it bold...